Classes/BackEnd/AbstractEventMailForm.php

```diff
@@ -703,11 +704,15 @@ protected function getHooks(): array
     protected function getRouteUrl(string $moduleName, array $urlParameters = []): string
     {
         $uriBuilder = $this->getUriBuilder();
-        try {
+        if (Typo3Version::isNotHigherThan(8)) {
+            try {
+                $uri = $uriBuilder->buildUriFromRoute($moduleName, $urlParameters);
+            } catch (\TYPO3\CMS\Backend\Routing\Exception\RouteNotFoundException $e) {
+                // no route registered, use the fallback logic to check for a module
+                $uri = $uriBuilder->buildUriFromModule($moduleName, $urlParameters);
+            }
+        } else {
             $uri = $uriBuilder->buildUriFromRoute($moduleName, $urlParameters);
```

Comment:
This does not seem to work in 9LTS. It looks like we need to build the route in another way.

Reply:
I guess we just need to load ext:backend in the test.